var/spack/repos/builtin/packages/rtmpdump/rtmpdump-fix-chunk-size.patch

```diff
@@ -0,0 +1,48 @@
+https://git.alpinelinux.org/aports/commit/main/rtmpdump/fix-chunk-size.patch?id=bf39fb1177ee77eee6c214a7393cc0054958ce08
+https://git.alpinelinux.org/aports/commit/main/rtmpdump/fix-chunk-size.patch?id=69bc162319b12e9b6c6d3ea345dbf7c218753594
```

Comment:
I guess it's fine, but out of curiosity: why are those ancient patches not upstreamed?

Reply:
I haven't investigated in detail, but I assume upstream is not responsive, considering these patches are carried by at least two distributions (Alpine and Gentoo).